### ISSUE TITLE
Using Kroki with attribute kroki-plantuml-include doesn't render preview

### ISSUE THREAD

**Issue by ahus1** [asciidoctor/asciidoctor-intellij-plugin#639]
## Observed vs. expected behavior

When using the attribute `kroki-plantuml-include`, the preview displays a broken image. 

Instead, I would have expected an image in the preview.

(discovered when testing examples from https://github.com/asciidoctor/asciidoctor-diagram/issues/335)

## Steps to reproduce

sample:
```
:kroki-plantuml-include: roles.puml

[plantuml]
----
rectangle checkout {
  customer -- (checkout)
  (checkout) .> (payment) : include
  (help) .> (checkout) : extends
  (checkout) -- clerk
}
----
```

roles.puml:
```
left to right direction
skinparam packageStyle rectangle
skinparam monochrome true
actor customer
actor clerk
```

## Environment

Plugin Version: 0.32.8

**Comment by ahus1** [comment]
Created pull request in upstream project: https://github.com/Mogztter/asciidoctor-kroki/pull/195

**Comment by ahus1** [comment]
This fixed in pre-release 0.32.9. 

The pre-release of the plugin is available from [GitHub releases](https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases) and the [IntelliJ AsciiDoc EAP repository](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/add-eap-repository-to-ide.html).
